src/components/invite/Invite.js

```diff
@@ -351,7 +365,8 @@ const Invite = () => {
           {`How Do I Invite People?`}
         </CustomButton>
       </View>
-      {showHowTo ? <InvitesHowTO /> : <InvitesData {...{ invitees, refresh, level }} />}
+      {showHowTo && <InvitesHowTO />}
+      <InvitesData {...{ invitees, refresh, level, totalEarned: get(inviteState, 'totalEarned', 0) }} />
```

Comment:
just add `const { totalEarned = 0 } = inviteState` before `return`
then just use `<InvitesData {...{ invitees, refresh, level, totalEarned}} />`

Reply:
done

Reply:
what if inviteState is null?
i've moved the get not to be inline